packages/frontend-shared/src/graphql/testMutation.ts

```diff
@@ -0,0 +1,32 @@
+import type { App, Mutation, Project } from '../generated/test-graphql-types.gen'
+import type { GraphQLResolveInfo } from 'graphql'
+
+import { app } from './testApp'
+
+type MaybeResolver<T> = {
+  [K in keyof T]: K extends 'id' | '__typename' ? T[K] : T[K] | ((args: any, ctx: object, info: GraphQLResolveInfo) => MaybeResolver<T[K]>)
+}
+
+export const mutation: MaybeResolver<Mutation> = {
```

Comment:
@lmiller1990 @tgriesser I need help connecting this. I tried a couple of ways but I keep getting this error. My test is calling the mutation, but I'm missing something that actually calls this. I checked out the CloudTypes because it seems to be doing something similar with query. 

![image](https://user-images.githubusercontent.com/8095629/135544772-25ddf625-351a-4fa9-8337-4002cc3eaecb.png)

Reply:
Taking a look